src/home_robot_sim/home_robot_sim/env/habitat_ovmm_env/habitat_ovmm_env.py

```diff
@@ -322,34 +352,64 @@ def _preprocess_action(
                 grip_action = 1
 
             turn = 0
+            forward = 0
             if action == DiscreteNavigationAction.TURN_RIGHT:
-                turn = -1
-            elif action in [
-                DiscreteNavigationAction.TURN_LEFT,
-            ]:
-                turn = 1
-
-            face_arm = (
-                float(action == DiscreteNavigationAction.MANIPULATION_MODE) * 2 - 1
-            )
+                turn = -self.discrete_turn_degrees
+            elif action == DiscreteNavigationAction.TURN_LEFT:
+                turn = self.discrete_turn_degrees
+            elif action == DiscreteNavigationAction.MOVE_FORWARD:
+                forward = self.discrete_forward
+
+            arm_action = np.zeros(self.joints_dof)
+            curr_joint_pos = self.get_current_joint_pos(habitat_obs)
+            target_joint_pos = curr_joint_pos
+            if action == DiscreteNavigationAction.MANIPULATION_MODE:
+                # turn left by 90 degrees, positive for turn left
+                turn = 90
+                target_joint_pos = curr_joint_pos.copy()
+                # current behavior
+                target_joint_pos[JointActionIndex.HEAD_PAN] = -1.7375  # look at ee
+                arm_action = target_joint_pos - curr_joint_pos
+                # TODO: robot config tries to go to state: STRETCH_PREGRASP_Q
+                # np.array([-np.pi / 2, -np.pi / 4])
+                # These serve as initial states of trained RL policies
+                # current sim:  [0.0, 0.775, 0.0, -1.57, 0.0, -1.7375, -0.7125]
+                # current real: [0.01, 0.6, 1.57, -1.51, 0.0, -1.57, -0.785] # STRETCH_PREGRASP_Q
+                # proposed    : [0.0, 0.775, 0.0, -1.57, 0.0, -1.57, -0.7125]  # STRETCH_PREGRASP_Q (new)
+                # [check best tilt for manip, check pan -1.57 vs pan -1.7375]
+                # [lift, arm, wrist yaw, wrist pitch, wrist roll, cam pan, cam tilt]
+            elif action == DiscreteNavigationAction.NAVIGATION_MODE:
+                # TODO: harcoded to maintain current behavior: use NAVIGATION_Q
+                target_joint_pos = np.array([0, 0.775, 0, -1.57000005, 0, 0.0, -0.7125])
```

Comment:
Remove hard coded pose?

Reply:
Yes, had them hardcoded with comments only to confirm the values because they were different in sim/real. Now updated PREGRASP_Q and NAVIGATION_Q